### ISSUE TITLE
AdminThemeUikit: Inconsistent red borders when displaying errors

### ISSUE THREAD

**Issue by fijha** [processwire/processwire-issues#593]
### Short description of the issue

Inconsistent red outline borders when displaying "Missing required value".

### Screenshots that demonstrate the issue

![border-1](https://user-images.githubusercontent.com/17672049/39957770-7fdc2848-55f8-11e8-9080-8888d9137162.png)

![border-2](https://user-images.githubusercontent.com/17672049/39957771-8003b958-55f8-11e8-8d0b-d2b72ea54dde.png)

### Suggestion for a possible fix

The simplest fix would be just to remove the red outline and show the default grey. I think it looks even better without it.

**Comment by Toutouwai** [comment]
The hidden outline relates to stacking order. One solution could be to give `.InputfieldStateError` some z-index CSS like: `position: relative; z-index: 2;`.

But if any adjacent inputfields have a special outline style (e.g. the "Warning" option in the AdminThemeUikit "Color" settings for the field) then it's inevitable that one inputfield's outline will be behind another.

**Comment by fijha** [comment]
Yes, It's problematic. `z-index` helps, but only in some cases, as you described. Probably the best would be to stick with grey border for all field color settings.

**Comment by ryancramerdesign** [comment]
I tried both options, and like the z-index option better in terms of appearance. The overlap isn't a problem if I choose z-index's appropriately (darker always on top of lighter, like for errors). But what I'm worried about is potential side effects of adding relative/z-index, and what happens when Inputfield layouts get more complex. So the common online color (grey) just seems to be the safest bet. For now, that's what I've added, per the suggestion. But I may come back to revisit this later as I think the state-specific outline color helps a lot in terms of directing the eye.
